srml/staking/src/lib.rs

```diff
@@ -43,6 +43,47 @@ const DEFAULT_MINIMUM_VALIDATOR_COUNT: u32 = 4;
 const MAX_NOMINATIONS: usize = 16;
 const MAX_UNSTAKE_THRESHOLD: u32 = 10;
 
+// a wrapper around validation candidates list and some metadata needed for election process.
```

Comment:
Phragmén specific data structures should be in their own module along with the algorithm.

Reply:
Maybe check the new function signature etc. and see if it is ok.